dev-utils/test.js

```diff
@@ -225,36 +230,12 @@ function runE2eTests(configFilePath, runSelenium) {
   }
 }
 
-function getConfig() {
-  const env = getTestEnvironmentVariables()
-  let serverUrl = 'http://localhost:8200'
-  if (env.serverUrl) {
-    serverUrl = env.serverUrl
-  }
-
-  const config = {
-    agentConfig: {
-      serverUrl,
-      serviceName: 'apm-agent-js-base/test'
-    },
-    useMocks: false,
-    mockApmServer: false,
-    serverUrl,
-    env
-  }
-  // if (env.sauceLabs) {
-  //   config.useMocks = true
-  // }
-  return config
-}
-
 module.exports = {
-  runUnitTests,
```

Comment:
I think these scripts also give us more control over how to run the tests!

Reply:
With the launchSaunceconnect manul option removed. this script would simply be an alias to runKarma which we had before. It doesn't really feel needed IMO.